packages/components/src/components/combobox/combobox.styles.ts

```diff
@@ -0,0 +1,344 @@
+
+  .combobox__listbox ::slotted(syn-divider) {
+    --spacing: var(--syn-spacing-x-small);
+  }
+
+  .combobox__listbox ::slotted(small) {
+    display: block;
+    font-size: var(--syn-font-size-small);
+    font-weight: var(--syn-font-weight-semibold);
+    color: var(--syn-color-neutral-500);
+    padding-block: var(--syn-spacing-2x-small);
+    padding-inline: var(--syn-spacing-x-large);
+  }
+
+
+  /**
+   * Invalid user data
+   */
+  :host([data-user-invalid]) .combobox__inputs {
+    border-color: var(--syn-input-border-color-focus-error);
+  }
+
+  :host([data-user-invalid]) .combobox--standard:not(.combobox--disabled).combobox--open .combobox__inputs,
+  :host([data-user-invalid]) .combobox--standard:not(.combobox--disabled).combobox--focused .combobox__inputs {
+    border-color: var(--syn-input-border-color-focus-error);
+    box-shadow: 0 0 0 var(--syn-focus-ring-width) var(--syn-input-focus-ring-error);
+  }
+
+  /* Clear button */
+  .combobox__clear {
+    color: var(--syn-input-icon-icon-clearable-color);
+    font-size: var(--syn-spacing-large);
+  }
+
+  .combobox--small .combobox__clear {
+    font-size: var(--syn-spacing-medium);
+  }
+
+  .combobox--large .combobox__clear {
+    font-size: var(--syn-spacing-x-large);
+  }
+
+  .combobox--medium .combobox__clear {
+    margin-inline-start: var(--syn-spacing-small);
+  }
+
+  /* Expand icon */
+  .combobox__expand-icon {
+    color: var(--syn-color-neutral-950);
+    font-size: var(--syn-spacing-large);
+    margin-inline-start: var(--syn-spacing-small);
+  }
+
+  /* Change combobox border on hover */
+  .combobox:not(.combobox--disabled):hover .combobox__inputs {
+    border-color: var(--syn-input-color-hover);
+  }
+
+  /* Prefix and Suffix */
+
+  /* Small */
+  .combobox--small .combobox__prefix::slotted(*) {
+    margin-inline-end: var(--syn-spacing-x-small);
+  }
+
+  .combobox--small .combobox__suffix::slotted(*) {
+    margin-inline-start: var(--syn-spacing-x-small);
+  }
+
+  .combobox--medium .combobox__prefix::slotted(*) {
+    margin-inline-end: var(--syn-input-spacing-small);
+  }
+
+  .combobox--medium .combobox__suffix::slotted(*) {
+    margin-inline-start: var(--syn-input-spacing-small);
+  }
+
+  .combobox--large .combobox__prefix::slotted(*) {
+    margin-inline-end: var(--syn-input-spacing-medium);
+  }
+
+  .combobox--large .combobox__suffix::slotted(*) {
+    margin-inline-start: var(--syn-input-spacing-medium);
+  }
+
+  .combobox__prefix,
+  .combobox__suffix {
+    color: var(--syn-input-help-text-color);
+  }
+
+  /* Listbox */
+  .combobox__listbox {
+    /* @todo: Should be --syn-border-radius-medium, which should be set to 0 */
+    border-radius: 0;
+    box-shadow: var(--syn-shadow-medium);
+  }
+
+  /**
+   * Make sure to hide the syn-divider for the first syn-optgroup
+   * Note! ::slotted does currently not work with ::part, so we
+   * opted for using a css variable here.
+   */
+  .combobox__listbox ::slotted(syn-optgroup:first-of-type) {
```

Comment:
I think this can be removed as it is specific to `syn-select[multiple]`.

Reply:
Just seen there is a demo for this indeed. Thats quite nice I think, ignore this comment :)